Java/core/src/main/java/com/amazon/randomcutforest/store/SmallIndexManager.java

```diff
@@ -75,6 +93,39 @@ public SmallIndexManager(short[] freeIndexes, short freeIndexPointer) {
         }
     }
 
+    public SmallIndexManager(int capacity, short[] freeIndexes, short freeIndexPointer) {
```

Comment:
Do we still need this constructor? We should delete unnecessary constructors before they get baked into the public API.

Reply:
This should be limited to the mapper classes. The bitset constructor makes sense when the external entity does not have information about the stack. The stack and the pointer still saves over the bit set in the serialized model. But it may be worth looking at this option once the entirety of changes to the mappers are in.